Build/toolchain/terapines.mk

```diff
@@ -16,20 +16,20 @@ endif
 ifneq ($(findstring newlib,$(STDCLIB)),)
 ### Handle cases when STDCLIB variable has newlib in it
 ifeq ($(STDCLIB),newlib_full)
-LDLIBS += -lc_size -lclang_rt.builtins
+LDLIBS += -lc -lclang_rt.builtins
```

Comment:
Do you have a size version for c library, if yes, you can use c_size version.

Reply:
Now ZCC no longer needs to use the _size suffix to search for the size version of the library. ZCC will go to the size subdirectory to find the size version of the library based on the Os flag.

Reply:
Cool